lib/bin/run_test.js

```diff
@@ -63,6 +63,23 @@ var _ = require('lodash');
 var co = require('co');
 var exit = require('exit');  // process.exit that works on Windows
 var promiseUtil = require('../promise_util');
+var cleaned = false;
+
+function cleanup(exitCode) {
+  if (cleaned) {
+    return;
+  }
+  cleaned = true;
+  require('ps-tree')(process.pid, function (err, children) {
```

Comment:
According to nodejs doc, require is caching and fetching the same object.. thus saving a line of code:
> Modules are cached after the first time they are loaded. This means (among other things) that every call to require('foo') will get exactly the same object returned, if it would resolve to the same file.